source/services/telegram.js

```diff
@@ -5,7 +5,7 @@
  */
 
 module.exports = {
-    popupUrl: 'https://telegram.me/share/url?url={url}',
+    popupUrl: 'https://telegram.me/share/url?url={url}&text={text}',
```

Comment:
Is `text={text}` necessary? As far as I understand from [`LikelyButton#addAdditionalParamsToUrl`](https://github.com/ilyabirman/Likely/blob/master/source/button.js#L214), there’s no need to specify this parameter, it’ll be appended automatically.